shopinvader/services/cart.py

```diff
@@ -70,6 +72,10 @@ def clear(self):
         cart = self._clear_cart(cart)
         return self._to_json(cart)
 
+    def _add_item_create_cart_params(self, **params):
+        # Hook here to customize cart creation
```

Comment:
@sebastienbeau @hparfr is it better like this? I added only a hook and support passing params to _create_empty_cart. The rest is left to custom code.